pkg/restore/restore.go

```diff
@@ -700,6 +700,7 @@ func (ctx *context) restoreResource(resource, namespace, resourcePath string) (a
 		applicableActions = append(applicableActions, action)
 	}
 
+fileLoop:
```

Comment:
Is this supposed to be here?

Reply:
Yes. I have to break out of two levels of loop, and using a named label is the only way of doing it in go. The only other way of doing it would be a significant refactoring of this (long) function.